### PR TITLE
Bugfix

### PR DESCRIPTION
**Pull request by BranChiaa**

### CODE REVIEW

File: src/assets/styles/tailwind.css [hunk 1]
```diff
@@ -1599,6 +1599,10 @@ float: right;
   height: 600px;
 }
 
+.h-700-px {
```
Comment (kavxya) [inline src/assets/styles/tailwind.css]: inconsistent naming....

File: src/assets/styles/tailwind.css [hunk 2]
```diff
@@ -2600,6 +2604,10 @@ margin: 2.25rem;
   width: auto;
 }
 
+.w-1\/12{
```
Comment (kavxya) [inline src/assets/styles/tailwind.css]: Name accordingly to the standard formats!!

**Review by kavxya — CHANGES_REQUESTED** [review]

File: src/assets/styles/tailwind.css (hunk 1 above)
Comment (BranChiaa) [reply src/assets/styles/tailwind.css]: yes boss

**Review by kavxya — APPROVED** [review]
Looks amazing!!

**Pull request merged**